src/sql/parts/insights/browser/insightsDialogView.ts

```diff
@@ -77,6 +77,7 @@ function stateFormatter(row: number, cell: number, value: any, columnDef: Slick.
 
 export class InsightsDialogView extends Modal {
 
+	private _table
```

Comment:
should this variable have a type?  And it looks like the `;` is missing.

Reply:
..I actually don't see where this is being used, so probably just deleting the line makes sense.

Reply:
Not sure how it got there, I'll remove it